### PR TITLE
Add parachute/flight termination docs

### PR DESCRIPTION
**Pull request by hamishwillee**
@bresch Thanks for information in https://github.com/PX4/Firmware/pull/10179#issuecomment-489828349 - that's an excellent starting point.

To structure this I have:
1. Added information about Failure Detector to the Safety page. From here I also linked to a new Flight Termination Config Page.
2. Added **Advanced Config > Flight Termination Config** page that generically explains how this works.
3. Added a peripherals  > Parachute page that offers specific information about working with parachutes and cross links to the others. It is less generic and has some duplication with the others, but I figure having a standalone page makes it easier to find.

Comments in line. 

**NOT TO BE MERGED UNTIL AFTER https://github.com/PX4/Firmware/pull/10179**

### CODE REVIEW

File: en/peripherals/parachute.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Motor settings:
+- Set [PWM_AUX_FAILn](../advanced_config/parameter_reference.md#PWM_AUX_FAIL1), where n is 1 - 4, to 900ms such that the motors directly go to disarmed values.
+
+
+> **Note** There is no way to recover from a Termination state!
+  Reboot/power cycle the vehicle before your next test.
+
+
+<!--
```
Comment (hamishwillee) [inline en/peripherals/parachute.md]: @bresch This param description is in master - seems incorrect "Does not affect the behavior of the vehicle for now; only for logging"

File: en/config/safety.md [hunk 2]
```diff
@@ -229,6 +231,21 @@ Parameter | Description
...
+The failure detector triggers [flight termination](../advanced_config/flight_termination.md) if the vehicle attitude ever exceeds a predefined attitude for more than a specified time (e.g. if it unexpectedly flips).
+This allows you to launch a parachute or perform some other action to protect the vehicle.
+
+The relevant parameters are shown below:
+
+Parameter | Description
+--- | ---
+<span id="FD_FAIL_P"></span>[FD_FAIL_P](../advanced_config/parameter_reference.md#FD_FAIL_P) | Maximum allowed pitch (in degrees).
+<span id="FD_FAIL_R"></span>[FD_FAIL_R](../advanced_config/parameter_reference.md#FD_FAIL_R) | Maximum allowed roll (in degrees).
+<span id="FD_FAIL_P_TTRI"></span>[FD_FAIL_P_TTRI](../advanced_config/parameter_reference.md#FD_FAIL_P_TTRI) | Time to exceed [FD_FAIL_P](#FD_FAIL_P) for failure detection (default 0.3s).
```
Comment (hamishwillee) [inline en/config/safety.md]: The PR is WIP because these two params aren't live yet.

File: en/config/safety.md [hunk 3]
```diff
@@ -229,6 +231,21 @@ Parameter | Description
 [VT_FW_ALT_ERR](../advanced_config/parameter_reference.md#VT_FW_ALT_ERR) | Maximum negative altitude error for fixed wing flight. If the altitude drops more than this value below the altitude setpoint the vehicle will transition back to MC mode and enter failsafe RTL.
 
 
+## Failure Detector {#failure_detector}
```
Comment (hamishwillee) [inline en/config/safety.md]: @bresch Does failure detector work on all vehicles and modes? What about acro modes, where things are supposed to turn over? How is it disabled for a particular vehicle?

Comment (bresch) [reply en/config/safety.md]: Unfortunately, the trigger is active as soon as the circuit-breaker is disabled. It's active for all modes, which means that someone flying in acro will trigger termination during a flip. The plan is to make further improvements after the Firmware PR will be merged (after 1.9 release).

File: en/peripherals/parachute.md (hunk 1 above)
Comment (bresch) [reply en/peripherals/parachute.md]: Yes, PR is not merged and the functionality does not exist on master yet

File: en/advanced_config/flight_termination.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Depending on what devices you have connected the PWM failsafe outputs can be used to:
+- Deploy a [parachute](../peripherals/parachute.md).
+- Extend retractable landing gear.
+- Move a PWM-connected gimbal to a safe orientation (or retract it) in order to protect the camera.
+- Trigger an inflatable device like an airbag.
+- Trigger an alarm.
+
+There is no way to recover from flight termination.
+After triggering you should unplug the battery as soon as possible
```
Comment (bresch) [inline en/advanced_config/flight_termination.md]:
```suggestion
After triggering you should unplug the battery as soon as possible.
```

File: en/advanced_config/flight_termination.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+These can be connected to any free PWM port (both MAIN and AUX).
+
+> **Note** If you're using Pixhawk-series board you will have to separately power the servo rail (i.e. from a 5V BEC, which is often also available from your ESC).
+
+
+## Software Configuration
+
+The [Safety](../config/safety.md) topic explains how to set *flight termination* as the action to be performed in the event of particular checks being triggered (this also covers the parameters you need to set to trigger the [Failure Detector](../config/safety.md#failure_detector), which always results in *Flight termination*).
+
+- Set [CBRK_FLIGHTTERM=0](../advanced_config/parameter_reference.md#CBRK_FLIGHTTERM) to enable flight termination.
```
Comment (bresch) [inline en/advanced_config/flight_termination.md]: Note that this only cuts FailureDetector and FMU loss features. The other flight terminations (RC, telem, geofence) are not disabled by that circuit breaker.

File: en/peripherals/parachute.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- Parachutes have a minimum effective altitude.
+- A parachute may trigger while the vehicle is upside down.
+  This will increase the time required to slow, and may even result in the drone collapsing the parachute.
+- The system requires a working and powered flight controller.
+  If something causes the flight stack to crash, the parachute will not deploy. 
+
+
+## Parachute Setup
+
+The setup below explains how you might configure a quad multicopter with motors on AUX 1-4 and a parachute on MAIN PWM port 5.
```
Comment (bresch) [inline en/peripherals/parachute.md]:
```suggestion
The setup below explains how you might configure a quad multicopter with motors on MAIN 1-4 and a parachute on MAIN PWM port 7.
```

File: en/peripherals/parachute.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+The PWM values mentioned are for the *Fruity Chutes* parachute we tested.
+
+> **Tip** For the first test, try on the bench, without the props and with an unloaded parachute device!
+
+Hardware setup:
+- Connect the parachute to IO, channel 5 (starting from 1).
```
Comment (bresch) [inline en/peripherals/parachute.md]:
```suggestion
- Connect the parachute to the IO port (MAIN), channel 7 (starting from 1).
```

File: en/peripherals/parachute.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+- Power the servo rail - i.e. connect a 5V BEC to the servo rail.
+
+Enable flight termination:
+- Set [CBRK_FLIGHTTERM](../advanced_config/parameter_reference.md#CBRK_FLIGHTTERM) to 0
+- Set [Safety](../config/safety.md) action to *Flight termination* for checks where you want the parachute to trigger
+- Set [Failure Detector](../config/safety.md#failure_detector) pitch and roll angles and time triggers for crash/flip detection. 
+
+Parachute settings
+- Set [PWM_MAIN_DIS5](../advanced_config/parameter_reference.md#PWM_MAIN_DIS5) to PWM value for parachute "OFF" position (usually between 700 and 1000ms)
```
Comment (bresch) [inline en/peripherals/parachute.md]:
```suggestion
- Set [PWM_MAIN_DIS7](../advanced_config/parameter_reference.md#PWM_MAIN_DIS5) to PWM value for parachute "OFF" position (usually between 700 and 1000ms)
```

File: en/peripherals/parachute.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+- Set [PWM_MAIN_FAIL5](../advanced_config/parameter_reference.md#PWM_MAIN_FAIL5) to PWM value for parachute "ON" position (usually between 1800 and 2200ms)
```
Comment (bresch) [inline en/peripherals/parachute.md]:
```suggestion
- Set [PWM_MAIN_FAIL7](../advanced_config/parameter_reference.md#PWM_MAIN_FAIL5) to PWM value for parachute "ON" position (usually between 1800 and 2200ms)
```

File: en/peripherals/parachute.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+
+Motor settings:
+- Set [PWM_AUX_FAILn](../advanced_config/parameter_reference.md#PWM_AUX_FAIL1), where n is 1 - 4, to 900ms such that the motors directly go to disarmed values.
```
Comment (bresch) [inline en/peripherals/parachute.md]:
```suggestion
- Set [PWM_MAIN_FAILn](../advanced_config/parameter_reference.md#PWM_MAIN_FAIL1), where n is 1 - 4, to 900ms such that the motors directly go to disarmed values.
```

File: en/config/safety.md (hunk 3 above)
Comment (hamishwillee) [reply en/config/safety.md]: Really? That's horrible to the point I'd call it a critical defect against v1.9.

File: en/advanced_config/flight_termination.md (hunk 5 above)
Comment (hamishwillee) [reply en/advanced_config/flight_termination.md]: Made this more clear

**Comment by hamishwillee** [comment]
Thanks @bresch ! I've added the diagram and made corrections in line with your comments. 
When do you think the PR will go in so I can merge this?

File: en/config/safety.md (hunk 3 above)
Comment (bresch) [reply en/config/safety.md]: Clarification: the circuit breaker is enabled by default; it means that the failure detector is disabled by default. A racer pilot would most likely never disable that circuit breaker, this is why I don't see that as a critical problem.
However, I'll still add a condition such that FD_FAIL_(P|R) = 0 or -1 deactivate the check.

Comment (hamishwillee) [reply en/config/safety.md]: Thank you for that clarification. I have updated this to make that clear. Makes a lot more sense now. 

I do find it a little odd that there is one circuit breaker for both attitude and IMU issues

**Comment by bresch** [comment]
https://github.com/PX4/Firmware/pull/10179 has been merged

**Comment by hamishwillee** [comment]
Done. Thanks @bresch !

**Pull request merged**